### PR TITLE
Duplicate columns in metadata.csv

### PR DESCRIPTION
**Pull request by Hwesta**
metadata.csvs may have duplicate columns. Store values in a list to handle this case.  Also, fix some typos.

### CODE REVIEW

File: src/MCPClient/lib/clientScripts/archivematicaCreateMETSMetadataCSV.py [hunk 1]
```diff
@@ -43,7 +51,7 @@ def parseMetadata(SIPPath):
     See parseMetadataCSV for details on parsing.
 
     :param SIPPath: Path to the SIP
-    :return: {<filename>: OrderedDict{<metadata name>=<metadata value>} }
+    :return: {<filename>: OrderedDict(key: [values]) }
```
Comment (mistydemeo) [inline src/MCPClient/lib/clientScripts/archivematicaCreateMETSMetadataCSV.py:55]: Nitpick, but document that it returns an `OrderedListsDict`?

Comment (Hwesta) [inline src/MCPClient/lib/clientScripts/archivematicaCreateMETSMetadataCSV.py:55]: I mostly used the OrderedListsDict so I didn't have to handle duplicates myself when inserting. Should I `return OrderedDict(metadata)` instead?

Comment (mistydemeo) [inline src/MCPClient/lib/clientScripts/archivematicaCreateMETSMetadataCSV.py:55]: Yeah, I'd say either document the actual return type or convert to an `OrderedDict` before returning.

**Comment by mistydemeo** [comment]
:thumbsup:

**Pull request merged**